src/app/backend/api_models.rs

```diff
@@ -214,6 +214,17 @@ impl Into<Vec<SongDescription>> for DetailedPlaylist {
     }
 }
 
+impl Into<Vec<SongDescription>> for Page<PlaylistTrack> {
```

Comment:
the api reference is really misleading, I just realized the generic `Tracks` used in this file in `DetailedPlaylist` is just a `Page` really, would you mind fixing that? this should allow  having only one conversion impl here (yours, for `Page<PlaylistTrack>`)

actually maybe we don't even need to have this tracks field at all on `DetailedPlaylist` since we no longer ask for the tracks field in `get_playlist_no_cache`!

Reply:
I'm using Playlist as the first conversion as it doesn't have a tracks variable and then returning DetailedPlaylist as it has tracks so I can add the tracks to it. I'm not really sure what you mean by this.

Reply:
Sorry, was thinking as I was typing! 

I re-took a look: the DetailedPlaylist type is no longer useful I believe, and can be removed!